app/src/main/java/org/simple/clinic/overdue/AppointmentRepository.kt

```diff
@@ -152,8 +152,12 @@ class AppointmentRepository @Inject constructor(
     )
   }
 
-  private fun transformSearchInputsIntoQuery(searchInputs: List<String>) = searchInputs
-      .joinToString(separator = " OR ") { "*$it*" }
+  private fun transformSearchInputsIntoQuery(searchInputs: List<String>): String {
```

Comment:
This should handle the overdue search, can we also check if this issue is present in the patient search as well?

Reply:
@msasikanth For the patient search, only alphaumeric values are allowed. So the bug won't arise over there. Either we remove the alphanumeric restriction and then transform the search input or we leave it as it is. Waiting for your feedback.